src/components/views/location/LocationPicker.tsx

```diff
@@ -289,16 +289,25 @@ export function getGeoUri(position: GeolocationPosition): string {
     const lat = position.coords.latitude;
     const lon = position.coords.longitude;
     const alt = (
-        position.coords.altitude !== undefined
+        isNumeric(position.coords.altitude)
             ? `,${position.coords.altitude}`
             : ""
     );
     const acc = (
-        position.coords.accuracy !== undefined
+        isNumeric(position.coords.accuracy)
             ? `;u=${ position.coords.accuracy }`
             : ""
     );
     return `geo:${lat},${lon}${alt}${acc}`;
 }
 
+function isNumeric(n: number | null | undefined) {
+    return (
+        n !== null &&
+        n !== undefined &&
+        !isNaN(n) &&
+        isFinite(n)
+    );
+}
```

Comment:
`Number.isFinite`? 

![image](https://user-images.githubusercontent.com/1190097/146967486-5fdf4484-faa0-4c0b-97e8-fe34ba9006a6.png)

Reply:
Thanks, much better.  Fixed in 08e80e1c5e